automated_packaging/prepare_changelog.pl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/perl
+use lib '/usr/local/bin';
+use common_functions;
+use JSON;
+
+#!usr/bin/perl
+$NEW_VERSION = $ARGV[0];
+$PROJECT = $ARGV[1];
```

Comment:
Let's switch order of parameters so that project comes first. (did not check other scripts/functions but if applicable change order there too)